stable/spring-cloud-data-flow/templates/metrics-deployment-rabbit.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: extensions/v1beta1
```

Comment:
Please update to `apps/v1` and add a `spec.selector`.

Reply:
updated